### ISSUE TITLE
Error in importing CCLE 2019 cohort to cbio docker

### ISSUE THREAD

**Issue by ShixiangWang** [cBioPortal/cbioportal#9857]
The cohort zipped file is downloaded from https://www.cbioportal.org/ and unzipped into `study` directory.

Output:

```sh
ERROR: data_sv.txt: lines [31, 129, 141, (494 more)]: No Entrez gene id or gene symbol provided for site 1 and site 2. This record will not be loaded
WARNING: data_sv.txt: lines [569, 575, 577, (149 more)]: Gene symbol maps to a single Entrez gene id, but is also associated to other genes as an alias. The system will assume the official gene symbol to be the intended one.; values encountered: ['FLI1', 'MET', 'RPN2', '(30 more)']
INFO: data_sv.txt: Validation of file complete
INFO: data_sv.txt: Read 26729 lines. Lines with warning: 4908. Lines with error: 497

INFO: -: Validation complete


#######################################################################
One or more errors reported above. Please fix your files accordingly
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Command:

```sh
$ docker-compose run -v $PWD/Report/:/report -v $PWD/portalInfo/:/portalInfo:ro cbioportal metaImport.py -p /portalInfo -s /study/ccle_broad_2019 -o 
```

**Comment by ShixiangWang** [comment]
This also fails for cohort PCAWG

```sh
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Reading data from:  /study/pancan_pcawg_2020/data_mirna.txt
Recaching...
Finished recaching...
--> profile id:  24
--> profile name:  miRNA expression (UQ normalized)
--> genetic alteration type:  MRNA_EXPRESSION
--> total number of samples: 749
--> total number of data lines:  1864
--> records inserted into `sample_profile` table: 749
--> total number of data entries skipped (see table below):  1864
org.mskcc.cbio.portal.dao.DaoException: Something has gone wrong!  I did not save any records to the database!
at org.mskcc.cbio.portal.scripts.ImportTabDelimData.importData(ImportTabDelimData.java:307)
at org.mskcc.cbio.portal.scripts.ImportProfileData.run(ImportProfileData.java:122)
at org.mskcc.cbio.portal.scripts.ConsoleRunnable.runInConsole(ConsoleRunnable.java:145)
at org.mskcc.cbio.portal.scripts.ImportProfileData.main(ImportProfileData.java:147)

Warnings / Errors:
-------------------
0.  Entrez_Id null not found. Record will be skipped for this gene.; 1864x

ABORTED!
java.lang.RuntimeException: org.mskcc.cbio.portal.dao.DaoException: Something has gone wrong!  I did not save any records to the database!
at org.mskcc.cbio.portal.scripts.ImportProfileData.run(ImportProfileData.java:127)
at org.mskcc.cbio.portal.scripts.ConsoleRunnable.runInConsole(ConsoleRunnable.java:145)
at org.mskcc.cbio.portal.scripts.ImportProfileData.main(ImportProfileData.java:147)
Caused by: org.mskcc.cbio.portal.dao.DaoException: Something has gone wrong!  I did not save any records to the database!
at org.mskcc.cbio.portal.scripts.ImportTabDelimData.importData(ImportTabDelimData.java:307)
at org.mskcc.cbio.portal.scripts.ImportProfileData.run(ImportProfileData.java:122)
... 2 more
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
Error occurred during data loading step. Please fix the problem and run this again to make sure study is completely loaded.
Traceback (most recent call last):
  File "/usr/local/bin/metaImport.py", line 202, in <module>
    cbioportalImporter.main(args)
  File "/cbioportal/core/src/main/scripts/importer/cbioportalImporter.py", line 533, in main
    process_directory(jvm_args, study_directory, args.update_generic_assay_entity)
  File "/cbioportal/core/src/main/scripts/importer/cbioportalImporter.py", line 368, in process_directory
    import_study_data(jvm_args, meta_filename, data_filename, update_generic_assay_entity, study_meta_dictionary[meta_filename])
  File "/cbioportal/core/src/main/scripts/importer/cbioportalImporter.py", line 162, in import_study_data
    run_java(*args)
  File "/cbioportal/core/src/main/scripts/importer/cbioportal_common.py", line 985, in run_java
    raise RuntimeError('Aborting due to error while executing step.')
RuntimeError: Aborting due to error while executing step.
```

**Comment by dippindots** [comment]
`ccle_broad_2019` is been verified to be imported correctly using cBioPortal version `v5.3.6`. 
Please try our latest update to import again. https://github.com/cBioPortal/cbioportal-docker-compose/tree/master#run-cbioportal-using-docker-compose
Please feel free to ask questions at here: https://groups.google.com/g/cbioportal

**Comment by cdupai-bbi** [comment]
@ShixiangWang (and for anyone else googling) for pancan_pcawg_2020, I ran into similar issues and it was caused by the miRNA files data_mirna.txt and data_mirna_zscores.txt having non-HUGO IDs in the HUGO_ID column. It is a similar issue to this comment where the backend is tossing out any data without a valid HUGO_ID, you end up with no data, and it throws an error.: https://github.com/cBioPortal/cbioportal/issues/8544#issuecomment-828269521. If you exclude those files from your upload it should work.
